### PR TITLE
[ISV-1097] Remove "entrypoints" module from ignored coverage config

### PR DESCRIPTION
**Pull request by haripate**

### CODE REVIEW

File: operator-pipeline-images/operatorcert/tekton.py [hunk 1]
```diff
@@ -85,7 +86,7 @@ class PipelineRun:
     # Markdown TaskRun template
     TASKRUN_TEMPLATE = "| {icon} | {name} | {start_time} | {duration} |"
 
-    def __init__(self, obj: dict, taskruns: list[TaskRun]) -> None:
+    def __init__(self, obj: dict, taskruns: Any) -> None:
```
Comment (Allda) [inline operator-pipeline-images/operatorcert/tekton.py:89]: Why is this needed?

File: operator-pipeline-images/tests/entrypoints/test_bundle_dockerfile.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+from unittest.mock import MagicMock, patch
+
+from operatorcert.entrypoints.bundle_dockerfile import generate_dockerfile_content
+
+
+@patch("operatorcert.entrypoints.bundle_dockerfile.setup_argparser")
```
Comment (Allda) [inline operator-pipeline-images/tests/entrypoints/test_bundle_dockerfile.py:6]: `setup_argparser` is not called in the function so the mock is not needed.

File: operator-pipeline-images/tests/entrypoints/test_bundle_dockerfile.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+) -> None:
+    args = MagicMock()
+    args.bundle_path = "demo.yml"
+
+    mock_annotations.return_value = {
+        "operators.operatorframework.io.bundle.manifests.v1": "demo1",
+        "operators.operatorframework.io.bundle.metadata.v1": "demo2",
+    }
+    dockerfile = generate_dockerfile_content(args)
+    assert dockerfile is not None
```
Comment (Allda) [inline operator-pipeline-images/tests/entrypoints/test_bundle_dockerfile.py:20]: Please check the content. Verifying that result is not None is a very basic test that doesn't prove the function is correct.

File: operator-pipeline-images/tests/entrypoints/test_create_container_image.py [hunk 4]
```diff
@@ -145,3 +204,36 @@ def test_remove_latest_from_previous_image(mock_put: MagicMock, mock_get: MagicM
...
+
+@patch("operatorcert.entrypoints.create_container_image.setup_argparser")
+@patch("operatorcert.entrypoints.create_container_image.check_if_image_already_exists")
+@patch(
+    "operatorcert.entrypoints.create_container_image.remove_latest_from_previous_image"
+)
+@patch("operatorcert.entrypoints.create_container_image.create_container_image")
+@patch("operatorcert.entrypoints.create_container_image.json.load")
+@patch("operatorcert.entrypoints.create_container_image.open")
+def test_main(
```
Comment (Allda) [inline operator-pipeline-images/tests/entrypoints/test_create_container_image.py:225]: Just curious... In a bunch of entry points, you ignore the main function with `pragma`. Why do you test this one and ignore others?

File: operator-pipeline-images/tests/entrypoints/test_download_test_results.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
+from unittest.mock import MagicMock, patch
+
+from operatorcert.entrypoints import download_test_results
+
+
+@patch("operatorcert.entrypoints.download_test_results.setup_argparser")
+@patch("operatorcert.entrypoints.download_test_results.download_test_results")
+@patch("operatorcert.entrypoints.download_test_results.store_results")
+def test_main(
```
Comment (Allda) [inline operator-pipeline-images/tests/entrypoints/test_download_test_results.py:9]: Same here.

File: operator-pipeline-images/tests/entrypoints/test_ocp_version_info.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
+from unittest.mock import MagicMock, patch
+
+from operatorcert.entrypoints import ocp_version_info
+
+
+@patch("operatorcert.entrypoints.ocp_version_info.setup_argparser")
+@patch("operatorcert.entrypoints.ocp_version_info.pathlib.Path")
+@patch("operatorcert.entrypoints.ocp_version_info.ocp_version_info")
+@patch("operatorcert.entrypoints.ocp_version_info.json")
```
Comment (Allda) [inline operator-pipeline-images/tests/entrypoints/test_ocp_version_info.py:9]: Mocking json should not be needed.

File: operator-pipeline-images/tests/entrypoints/test_reserve_operator_name.py [hunk 7]
```diff
@@ -97,3 +97,5 @@ def test_reserve_operator_name(mock_post) -> None:
             }
         ]
     }
+
+    reserve_operator_name.reserve_operator_name(args)
```
Comment (Allda) [inline operator-pipeline-images/tests/entrypoints/test_reserve_operator_name.py:101]: Can you add a check that verifies that `post` was called with the correct payload?

File: operator-pipeline-images/tests/entrypoints/test_create_container_image.py (hunk 4 above)
Comment (haripate) [reply operator-pipeline-images/tests/entrypoints/test_create_container_image.py:225]: I forgot to remove these tests for the main function. Initially, I started with writing the unit tests for the main function, then realized that it isn't needed. I will remove them.

File: operator-pipeline-images/tests/entrypoints/test_download_test_results.py (hunk 5 above)
Comment (haripate) [reply operator-pipeline-images/tests/entrypoints/test_download_test_results.py:9]: Here, this is the only function in the `download_test_results.py` file, so I thought it would be better to include that test.

File: operator-pipeline-images/tests/entrypoints/test_bundle_dockerfile.py (hunk 2 above)
Comment (haripate) [reply operator-pipeline-images/tests/entrypoints/test_bundle_dockerfile.py]: Removed. I forgot to remove it before.

File: operator-pipeline-images/operatorcert/tekton.py (hunk 1 above)
Comment (haripate) [reply operator-pipeline-images/operatorcert/tekton.py]: I was getting type errors because of that, but It may be because of the python lower version I am using.

File: operator-pipeline-images/tests/entrypoints/test_bundle_dockerfile.py (hunk 3 above)
Comment (haripate) [reply operator-pipeline-images/tests/entrypoints/test_bundle_dockerfile.py]: Updated.

File: operator-pipeline-images/tests/entrypoints/test_ocp_version_info.py (hunk 6 above)
Comment (haripate) [reply operator-pipeline-images/tests/entrypoints/test_ocp_version_info.py:9]: I tried it before, but it was throwing me TypeError.

File: operator-pipeline-images/tests/entrypoints/test_reserve_operator_name.py (hunk 7 above)
Comment (haripate) [reply operator-pipeline-images/tests/entrypoints/test_reserve_operator_name.py:102]: I added the check on the `post` method.

File: operator-pipeline-images/tests/entrypoints/test_ocp_version_info.py (hunk 6 above)
Comment (Allda) [reply operator-pipeline-images/tests/entrypoints/test_ocp_version_info.py:9]: That's because you need to return valid json from `ocp_version_info` mock.

Comment (Allda) [reply operator-pipeline-images/tests/entrypoints/test_ocp_version_info.py:9]: @haripate Any update to this?

Comment (haripate) [reply operator-pipeline-images/tests/entrypoints/test_ocp_version_info.py]: I was on PTO. Just updated it.

**Review by Allda — APPROVED** [review]
